src/test/groovy/graphql/schema/idl/SchemaPrinterTest.groovy

```diff
@@ -892,66 +894,66 @@ type Query {
 
         then:
         // args and directives are sorted like the rest of the schema printer
-        result == '''"Directs the executor to include this field or fragment only when the `if` argument is true"
-directive @include(
-    "Included when true."
-    if: Boolean!
-  ) on FIELD | FRAGMENT_SPREAD | INLINE_FRAGMENT
-
-"Directs the executor to skip this field or fragment when the `if`'argument is true."
-directive @skip(
-    "Skipped when true."
-    if: Boolean!
-  ) on FIELD | FRAGMENT_SPREAD | INLINE_FRAGMENT
+        result == '''directive @argDirective on ARGUMENT_DEFINITION
```

Comment:
directives are now sorted by name like all the other types were, hence this change to the test - same in most of the directive tests.  They are now sorted by directive name just like the other types were